### PR TITLE
fix(ctl): Skip creating operator-namespace when passing `--skip-release` in stack/demo command

### PR DESCRIPTION
**Pull request by Techassi**
Discovered by @sbernauer, brought forward in Slack chat.

### CODE REVIEW

File: rust/stackable-cockpit/src/platform/stack/spec.rs [hunk 1]
```diff
@@ -159,15 +159,9 @@ impl StackSpecV2 {
         release_list: ReleaseList,
         operator_namespace: &str,
         product_namespace: &str,
-        skip_release_install: bool,
     ) -> Result<(), StackError> {
         info!("Installing release for stack");
```
Comment (sbernauer) [inline rust/stackable-cockpit/src/platform/stack/spec.rs:163]: Please remove the "for stack" (you can only install a release with no stack)" and mention the release name

**Review by sbernauer — DISMISSED** [review]
LGTM % comment
In general the whole namespace creation is a bit scattered around (e.g. the release install function should ensure the operator namespaces exists, stacks and demos should not care about this) but should be fine ;)

**Comment by Techassi** [comment]
Yes I agree. I already added a TODO on #98 to unify/refactor the namespace creation.

File: rust/stackable-cockpit/src/platform/stack/spec.rs (hunk 1 above)
Comment (Techassi) [reply rust/stackable-cockpit/src/platform/stack/spec.rs]: Changed in baffb30

**Review by sbernauer — APPROVED** [review]

**Pull request merged**